packages/rpc/src/util.ts

```diff
@@ -0,0 +1,13 @@
+import { NEAR_RPC_URL } from "./constants";
+import superagent from "superagent";
+
+export const requestFromNearRpc = async (
+  body: Record<string, any>
+): Promise<Record<string, any>> => {
+  const res = await superagent
```

Comment:
is superagent really needed ? I mean pure fetch wont work?

Reply:
Pure fetch won't work inside NodeJS